src/view-backend.c

```diff
@@ -93,6 +96,13 @@ wpe_view_backend_set_input_client(struct wpe_view_backend* backend, const struct
     backend->input_client_data = client_data;
 }
 
+void
+wpe_view_backend_set_fullscreen_client(struct wpe_view_backend* backend, const struct wpe_view_backend_fullscreen_client* client, void* client_data)
+{
+    backend->fullscreen_client = client;
+    backend->fullscreen_client_data = client_data;
```

Comment:
Do you have any example on how to assert that check? I mean do you mean adding a assert/panic or similar.

Checking a similar function (wpe_view_backend_set_input_client) I don't see it having the check or the destroy_notify callback

Reply:
We don't have such checks in other functions, but it would be good to have them. I think a plain `assert(!backend->fullscreen_client)` should be enough here 🤔

Reply:
I think you might have forgotten to add the assertion here 😉